assets/sass/_images.scss

```diff
@@ -0,0 +1,133 @@
+@import "compass/utilities/general/clearfix";
+@import "mixins";
+@import "variables";
+
+$scrollable_bg_color: #f9f9f9;
+
+.scrollable_wide_image {
+  width: 900px;
+  margin: 0 #{($text_max_width - 900px) / 2};
+
+  img {
+    max-width: 100%;
+  }
+
+  @include if_narrower_than(940px) {
+    max-width: 800px;
+    margin: 0 -#{(800px - $text_max_width) / 2};
+  }
+
+  @include if_narrower_than(850px) {
+    max-width: 700px;
+    margin: 0 -#{(700px - $text_max_width) / 2};
```

Comment:
- Make sure these are all consistent with line 9
- Make sure images look okay between 850px and 650px